test/wrappers/set-protocol-v2/PerpV2LeverageModuleWrapper.spec.ts

```diff
@@ -0,0 +1,510 @@
+import { ethers, Wallet } from 'ethers';
+// import { BigNumber } from 'ethers/lib/ethers';
+import { Address } from '@setprotocol/set-protocol-v2/utils/types';
+import { ZERO,  } from '@setprotocol/set-protocol-v2/dist/utils/constants';
+// import { ADDRESS_ZERO, ONE_DAY_IN_SECONDS } from '@setprotocol/set-protocol-v2/dist/utils/contracts';
+
+import {
+  Blockchain,
+  // bitcoin,
+  ether,
+  usdc as usdcUnits,
+  // preciseDiv,
+  // preciseMul,
+  // toUSDCDecimals,
+  // leverUp
+} from '@setprotocol/set-protocol-v2/dist/utils/common';
+import DeployHelper from '@setprotocol/set-protocol-v2/dist/utils/deploys';
+import { SystemFixture } from '@setprotocol/set-protocol-v2/dist/utils/fixtures/systemFixture';
+import { PerpV2Fixture } from '@setprotocol/set-protocol-v2/dist/utils/fixtures/PerpV2Fixture';
+import {
+  PerpV2,
+  PerpV2LeverageModule,
+  DebtIssuanceMock,
+  StandardTokenMock,
+  SetToken,
+  PerpV2BaseToken
+} from '@setprotocol/set-protocol-v2/dist/utils/contracts';
+
+import PerpV2LeverageModuleWrapper from '@src/wrappers/set-protocol-v2/PerpV2LeverageModuleWrapper';
+import { expect } from '../../utils/chai';
+
+const provider = new ethers.providers.JsonRpcProvider();
+const blockchain = new Blockchain(provider);
+
+describe('PerpV2LeverageModuleWrapper', () => {
+  let owner: Address;
+  // let mockModule: Address;
+  let deployer: DeployHelper;
+
+  let perpLib: PerpV2;
+  let perpLeverageModuleWrapper: PerpV2LeverageModuleWrapper;
+  let perpLeverageModule: PerpV2LeverageModule;
+  let debtIssuanceMock: DebtIssuanceMock;
+  let setup: SystemFixture;
+  let perpSetup: PerpV2Fixture;
+
+  let vETH: PerpV2BaseToken;
+  let vBTC: PerpV2BaseToken;
+  let usdc: StandardTokenMock;
+
+  beforeAll(async () => {
+    [
+      owner,
+      // mockModule,
+    ] = await provider.listAccounts();
+
+    deployer = new DeployHelper(provider.getSigner(owner));
+    setup = new SystemFixture(provider, owner);
+
+    perpSetup = new PerpV2Fixture(provider, owner);
```

Comment:
@cgewecke Is this the right way to prepare the PerpV2Fixture? it seems like `perpSetup.clearingHouseConfig` == undefined, as well as `perpSetup.usdc` == undefined in unit tests, but Im not sure what I'm doing wrong